x/emissions/keeper/inference_synthesis/network_inference_builder.go

```diff
@@ -34,9 +35,18 @@ func NewNetworkInferenceBuilderFromSynthRequest(
 		return nil, errorsmod.Wrapf(err, "Error building palette from request")
 	}
 	return &NetworkInferenceBuilder{
-		ctx:     req.Ctx,
-		logger:  Logger(req.Ctx),
-		palette: palette,
+		ctx:                        req.Ctx,
+		logger:                     Logger(req.Ctx),
+		palette:                    palette,
+		nonce:                      req.Nonce,
+		inferences:                 []*emissions.WorkerAttributedValue{},
```

Comment:
Fwiw, the zero value of a slice is `nil`, which will still satisfy `exhauststruct`.  If you do it your way, it'll allocate some memory/`cap`that you might not use.  I'd advise waiting to `make(...)` slices like these until you need them and maybe even have the benefit of knowing how large to make them

Same goes for maps.  Zero value is `nil`

This is because slices are pointers to a struct under the hood, as are maps and channels.  Everything else is pretty much what it appears to be